layouts/_default/baseof.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html lang="{{ .Site.LanguageCode }}">
+<html lang="{{ .Site.LanguageCode }}" class="direction-{{ .Language.LanguageDirection | default "ltr" }}"> 
```

Comment:
Is there any special reason to use class instead of HTML attribute `dir`?

I was testing using the following code and seems to be fine:

```html
<html lang="{{ .Site.LanguageCode }}" dir="{{ default `ltr` .Language.LanguageDirection }}">
```

Reply:
No specific reason, just using the same code used in Papermod's PR.
I just tested it locally, and it worked fine, so I changed it.